prism/src/main/webapp/WEB-INF/web.xml

```diff
@@ -41,6 +41,15 @@
     </filter>
 
     <filter>
+        <filter-name>csrf</filter-name>
+        <filter-class>org.apache.falcon.security.FalconCSRFFilter</filter-class>
+        <init-param>
+            <param-name>methods-to-ignore</param-name>
+            <param-value>GET</param-value>
```

Comment:
What do you mean by default configuration? Do you mean startup.properties? That is what I'm going to add to this patch. Thanks, @ajayyadava .